server/game/Interfaces.ts

```diff
@@ -153,7 +182,7 @@ type ITriggeredAbilityAggregateWhenProps<TSource extends Card> = ITriggeredAbili
     aggregateWhen: (events: GameEvent[], context: TriggeredAbilityContext) => boolean;
 };
 
-type IOngoingEffectGenerator = (game: Game, source: Card, props: IOngoingEffectProps) => (OngoingCardEffect | OngoingPlayerEffect);
+export type IOngoingEffectGenerator = (game: Game, source: Card, props: IOngoingEffectProps) => (OngoingCardEffect | OngoingPlayerEffect);
```

Comment:
Do you mind moving this above the comment line on 170 with the other exported types